internal/lib/container_server.go

```diff
@@ -431,6 +440,8 @@ func configNetNsPath(spec *rspec.Spec) (string, error) {
 	return "", fmt.Errorf("missing networking namespace")
 }
 
+var ErrIsNonCrioContainer = errors.New("non CRI-O container")
```

Comment:
I thought there was a file where Crio errors were defined?  Should this be there or am I'm having a bad memory?

Reply:
Hm, we don't have that many dedicated errors, so I decided to keep the error nearby where it's actually used. Do you think that we should create a dedicated place for errors?

Reply:
If we don't have a dedicated error file, then don't worry about it for now.  However, I personally do prefer having a central common file for hard error definitions like this one.   Maybe something to consider for a follow up PR if @mrunalp is hip with the thought.